components/Ink/src/private/MDCSimpleInkLayer.h

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <QuartzCore/QuartzCore.h>
+
+@interface MDCSimpleInkLayer : CAShapeLayer
+
+@property(nonatomic, assign) BOOL startAnimActive;
+@property(nonatomic, assign) CGFloat endAnimDelay;
+@property(nonatomic, assign) CGFloat finalRadius;
+@property(nonatomic, assign) CGFloat initialRadius;
+@property(nonatomic, strong) UIColor *inkColor;
+
+- (void)start:(CGPoint)point;
```

Comment:
naming is hard.
Consider a more descriptive name here and for end.
startRipple?  startAnimation?
naming is hard.

Reply:
Renamed to startAnimation